Moderation-Policy.md

```diff
@@ -217,14 +217,14 @@ decision was made should be added to the moderation repository thread.
 
 ### Temporary and Indefinite Bans
 
-An Indefinite Ban is set for an unspecified period of time and may only be
-lifted for an individual through a simple majority vote of the the Moderation
-Team.
-
 A Temporary Ban is time limited, with the timeframe decided on by the Moderation
 Team at the time of issuing, depending on the severity of the issue. Recommended
 default options are 24-hour, 48-hour, and 7-day periods.
 
+An Indefinite Ban is set for an unspecified period of time and may only be
+lifted for an individual through a simple majority vote of the the Moderation
```

Comment:
Nit: There are two "the"s here. I see that it was already there, but now that it is moved around, I think we can do this small change.